source/cfp.rst

```diff
@@ -0,0 +1,107 @@
+Call For Proposals
+==================
+
+We are planning to use Google Forms for handling CFP proposals.
+
+Daniele has set up a form with the following questions:
+
+About you
+---------
+
+We'd like to know a little bit about you for our files. If you submit more than
+one proposal, you don't need to send all of these details again.
```

Comment:
I'm not sure how this works in practice, since the "More information about you" field is required.

Could we gather this information after we've selected the programme?  Or do you want this information to inform how we select the programme?